content/build/hydrobuild.md

```diff
@@ -114,40 +133,44 @@ $ docker buildx build --builder cloud-<org>-default \
   --push .
 ```
 
-> **Note**
->
-> If you build multi-platform images, you won't be able to load the images back
-> to your local image store unless you turn on the containerd image store
-> feature, and use the `--load` flag.
->
-> Using the containerd image store with Hydrobuild currently results in slower
-> transfers of build output to the client, compared to when you use the default
-> image store.
->
-> When building multi-platform images, consider pushing the resulting image to
-> a registry directly, using the `docker buildx build --push` flag.
+If you want to build with a tag, but you don't want to load the results to your
+local image store, you export the build results to the build cache only:
 
-### Use by default
+```console
+$ docker buildx build --builder cloud-<org>-default \
+  --platform linux/amd64,linux/arm64 \
+  --tag <org>/<image> \
+  --output type=cacheonly .
+```
 
-If you want to use Hydrobuild by default, you can run the following command to
-make it the selected builder:
+## Multi-platform builds
+
+To run multi-platform builds, you must specify all of the platforms that you
+want to build for using the `--platform` flag.
 
 ```console
-$ docker buildx use cloud-<org>-default --global
+$ docker buildx build --builder cloud-<org>-default \
+  --platform linux/amd64,linux/arm64 \
+  --tag <org>/<image> \
+  --push .
 ```
 
-> **Note**
->
-> Changing your default builder with `docker buildx use` only changes the
-> default builder for the `docker buildx build` command. The `docker build`
-> command still uses the `default` builder, unless you specify the `--builder`
-> flag explicitly.
->
-> If you use build scripts, such as `make`, we recommend that you update your
-> build commands from `docker build` to `docker buildx build`, to avoid any
-> confusion with regards to builder selection. Alternatively, you can run
-> `docker buildx install` to make the default `docker build` command behave
-> like `docker buildx build`, without discrepancies.
+If you don't specify the platform, Hydrobuild automatically builds for the
+architecture matching your local environment.
+
+When building multi-platform images, you won't be able to load the images back
```

Comment:
@dvdksn did you try this ? I don't think it's even possible 🤔

Reply:
I think I assumed that the export worked like the docker-container driver 🤔